packages/manager/core/generator/app/templates/src/pages/error.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import {
+  OsdsMessage,
+  OsdsText,
+  OsdsButton,
+} from '@ovhcloud/ods-stencil/components/react/';
+import { OdsThemeColorIntent } from '@ovhcloud/ods-theming';
+import {
+  OdsMessageType,
+  OdsTextLevel,
+  OdsButtonVariant,
+} from '@ovhcloud/ods-core';
+import OOPS from '../assets/error-banner-oops.png';
+import './error.scss';
+
+export const DEFAULT_ASSET = OOPS;
```

Comment:
Where is it used ?

Reply:
From original Angular code: packages/manager/modules/error-page/src/error.constants.js